src/jshint.js

```diff
@@ -4378,6 +4378,10 @@ var JSHINT = (function() {
     } else {
       // ImportClause :: NamedImports
       advance("{");
+      // Breaking change
```

Comment:
> It would be good to have this in and then a undoing commit - we can then revert the undoing for v3

Why just don't open a new pr that will be merged for v3? So in the changelog there won't be both `Warn when an import is empty` and `Don't warn when an import is empty`, which is confusing.